DOCS-107 - CIS for AWS CloudQuery - onboarding improvements (#4209)
SumoLogic/sumologic-documentation: docs/security/cloud-infrastructure-security/cloud-infrastructure-security-for-aws.md

     * View Monitors
     * Manage Monitors
* **Sumo Logic Access ID and Key**. When you deploy the solution, you’ll need to supply a Sumo Logic [Access ID and Access Key](/docs/manage/security/access-keys/), which enable you to use Sumo Logic APIs. Make sure you have the role capabilities listed above before generating the Access ID and Key.

> * **AWS credentials**. To deploy the solution, you will need to log onto the [AWS Console](https://console.aws.amazon.com/console). For the CloudFormation template deployment, your AWS role must have the permissions described by [this JSON file](https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/AWSCISCFTemplatePermissions.json). As necessary, you may add JSON text to an existing or a new policy associated with an AWS IAM role as described in the [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-create-and-attach-iam-policy.html). You can set this up during installation. For more information, see [Check AWS role permission](#check-aws-role-permission).

Reviewer: Anchor link broken. Should it be #create-new-source-check-aws-role-permission?

Author: Yes! It's fixed now.

    * If you select **Use Existing Source**, select the source from the dropdown. <br/><img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-install-0a.png')} alt="Use Existing Source selection" style={{border: '1px solid gray'}} width="400"/>
    * If you select **Create New Source**, type the name you want to use for the source. <br/><img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-install-0b.png')} alt="Create New Source selection" style={{border: '1px solid gray'}} width="400"/>
1. Click **Next**. <br/>Depending on what you selected, one of the following occurs:
    * If you selected only **Use Existing Source** or **Do Not Collect**, the solution is installed. Proceed to [Start using the solution](#start-using-the-solution).

       :::tip
       Using only existing sources is the easiest way to install, since it means you don't have to create any new sources.
       :::

    * If you selected **Create New Source** for *any* source, the following screen appears. You must perform the steps in the following sections:
       * [Create new source: Select region](#create-new-source-select-region)

>        * [Create new source: Check AWS role permission](#create-new-source-check-aws-role-permission)

Reviewer: To match the casing in the UI screenshot

```suggestion
       * [Create new source: Check AWS Role Permission](#create-new-source-check-aws-role-permission)
```

       <details>
       <summary>CloudQuery</summary>

>        In this section, you have the choice to create a [CloudQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cloudquery-source) in Sumo Logic. 

Reviewer: To clarify - does this mean user has their choice of any CloudQuery source? Or is this a completely optional step altogether?

Author: It's an optional step.

Reviewer: Oh ok, thank you for clarifying. I'd maybe edit it like....

```suggestion
       This section demonstrates how you can optionally create a [CloudQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cloudquery-source) in Sumo Logic. 
```

Author: Sounds good!

          * **Name of existing S3 Bucket which contains the Network Firewall Logs**. If you selected **Yes** in the preceding field in this section for creating an S3 bucket, leave this blank. If you selected **No** in the preceding field for creating an S3 bucket, provide an existing S3 Bucket name which contains Network Firewall Logs. <br/><img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-param-7a.png')} alt="Firewall configuration" style={{border: '1px solid gray'}} width="700"/>
       </details>

> 1. Under **Permissions**, in **IAM role - optional**, choose the IAM role for CloudFormation to use for all operations performed on the stack. The role must have permissions to set up the necessary Lambdas, S3 buckets, Kenesis streams, and other objects needed in the CloudFormation template, as well as access to the appropriate logs. If your AWS role does not have the necessary permissions, see [Check AWS role permission](#check-aws-role-permission). <br/><img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-permissions.png')} alt="Create Stack button" style={{border: '1px solid gray'}} width="700"/>

Reviewer: Anchor link broken. Should it be #create-new-source-check-aws-role-permission?

Author: Yes! It's fixed now.

Author: Yes! I just fixed it.

Thanks for checking my latest updates and catching the errors, @kimsauce! You're the best!

Reviewer: No prob - you always do the same for me! I gotchu!🥳
